lib/vdsm/virt/backup.py

```diff
@@ -150,6 +150,9 @@ def __init__(self, backup_config):
 
 def start_backup(vm, dom, config):
     backup_cfg = BackupConfig(config)
+    log.info("BackupConfig object (s): %s", backup_cfg)
+    log.info("BackupConfig object (r): %r", backup_cfg)
```

Comment:
All the info is already logged via the API layer. I guess that this is a debugging
log for development?

Reply:
This is just to verify that the new VDSM version was installed.
These logs will help me to verify that :)